deployment/live/gcp/ci/terragrunt.hcl

```diff
@@ -4,6 +4,7 @@ terraform {
 
 locals {
   env                 = "ci"
+  docker_env          = local.env
```

Comment:
Could you use env directly?

Reply:
`docker_env` is required as the module is shared in both conformance CI (`env = "ci"`) and cloudbuild (`env = "prod"`). Keeping the `docker_env` is easier to understand at the moment.